src/Content/Feature.php

```diff
@@ -23,13 +23,15 @@ public static function isEnabled($uid, $feature)
 	{
 		$x = Config::get('feature_lock', $feature, false);
 
-		if ($x === false) {
+		if (is_null($x)) {
```

Comment:
This is counter-productive because we defined a default value in the `Config::get()` call, so the return value will never be `null`.